docs/extensions/index.mdx

```diff
@@ -0,0 +1,208 @@
+
+### How It Works
+
+1. During initialization, clients declare supported extensions in the `extensions` field
+2. Servers respond with their own supported extensions in their `extensions` field
+3. Each extension specification defines whether support from both parties is required
+4. Extensions may include optional configuration parameters
+
+### Example: Client Initialization with Extensions
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "initialize",
+  "params": {
+    "protocolVersion": "2025-11-25",
+    "capabilities": {
+      "roots": {
+        "listChanged": true
+      },
+      "extensions": {
+        "example.com/custom-extension": {
+          "exampleSetting": "value"
+        },
+        "modelcontextprotocol.io/official-extension": {}
+      }
+    },
+    "clientInfo": {
+      "name": "ExampleClient",
+      "version": "1.0.0"
+    }
+  }
+}
+```
+
+### Example: Server Response with Extensions
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": {
+    "protocolVersion": "2025-11-25",
+    "capabilities": {
+      "logging": {},
+      "extensions": {
+        "modelcontextprotocol.io/official-extension": {
+          "serverConfig": "enabled"
+        }
+      }
+    },
+    "serverInfo": {
+      "name": "ExampleServer",
+      "version": "1.0.0"
+    }
+  }
+}
+```
+
+### Extension Configuration
+
+Each extension in the `extensions` object can have an associated configuration object:
+
+- **Empty object `{}`**: Indicates support for the extension with default settings
+- **Configuration object**: Provides extension-specific settings and parameters
+- **Omission**: Extension is not supported
+
+Implementations SHOULD:
+
+- Ignore extensions they don't recognize
+- Gracefully handle missing configuration parameters
+- Validate configuration values before using them
+
+For detailed protocol-level documentation on capability negotiation, see the [Lifecycle documentation](/specification/draft/basic/lifecycle#extension-negotiation).
+
+## Extension Repositories
+
+Extension repositories are official repositories within the [modelcontextprotocol GitHub organization](https://github.com/modelcontextprotocol) with the `ext-` prefix. Each repository focuses on a specific area and is maintained by appointed maintainers.
+
+| Repository                                                   | Description                                         | Maintainers                                                                                 |
+| ------------------------------------------------------------ | --------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| [ext-auth](https://github.com/modelcontextprotocol/ext-auth) | Authentication and authorization extensions for MCP | [MAINTAINERS.md](https://github.com/modelcontextprotocol/ext-auth/blob/main/MAINTAINERS.md) |
```

Comment:
Is this table supposed to live here?  It seems redundant with the extensions registry page.

Reply:
> This page is more for developers that want to know how things are specified and where things live.

~Could you elaborate a bit?  I'm not sure I understand the difference.~

Oh, correlating with your other comment, these repositories are not for a single extension.

I was thinking each extension would have its own repository.